lib/public/components/common/popover/dropdown.js

```diff
@@ -37,43 +37,9 @@ export const dropdown = (
         trigger,
         h('.dropdown', content),
         {
+            ...PopoverPreConfiguration.click,
```

Comment:
`.click` was not clear for me at first glance, I would name it, maybe, `.clickBased`, something like in related docs